content/v2/tech/jupyterhub/intro.md

```diff
@@ -0,0 +1,21 @@
+# JupyterHub Overview
+
+This section describes the process of configuring and deploying your own Jupyterhub. It gives an overview of how to make important design decisions, such as where to deploy your hub and what Hub size is needed. 
+
+## Why JupyterHub?
+
+[**JupyterHub**](https://zero-to-jupyterhub.readthedocs.io/en/latest/) is a multi-user Hub that allows students to interact with their own computing environment. It makes it easy to spawn and manage multiple instances of a Jupyter notebook.
+
+Data 8 operates on a scale and level of complexity that required several technological improvements to our current teaching toolset. The Data 8 team worked closely with various open-source projects in order to contribute to tools that are broadly useful.
```

Comment:
Some statistics to add here: Data 8 started from Fall 2015 w/ 60 students to now with Spring 2020 having 1350 students. This shows how much we valued scaling our infrastructure to incorporate this.